### ISSUE TITLE
Feature Request: Backup and restore workspace config.

### ISSUE THREAD

**Issue by rahulhaque** [DevAlien/workspaces#58]
Is there any way I can backup my config and restore? Any workaround or is it planned?

Btw, thanks for all the effort put behind this awesome tool.

**Comment by DevAlien** [comment]
The config is stored in your home directory. Below .config if I remember
correctly. It is a simple json file. So you can save it and then just put
it in the same directory in another install.

I can add a button to make a backup and an import method.

Or I could also add a way to store the settings online if that's something
needed. Let me know

**Comment by rahulhaque** [comment]
@DevAlien a simple export/import button would be great for new users. 

I found the config file in `.local/share/com.github.devalien.workspaces` directory. I am running Elementary OS 5.1.7 Hera.

Also if you could have a look at this issue #55 I am facing. I can use it as SSH manager by leveraging this feature.

**Comment by rahulhaque** [comment]
![image](https://user-images.githubusercontent.com/12048213/122640261-6c3c9380-d120-11eb-9599-e56a19fb54ee.png)

Just an idea 😅️ you can add an option for SSH where user enters the username and host which is the simplest approach to connect to remote servers. Which will open a terminal prompting for password.

One tool for everything. Love this and keep up the good work. 👏️

**Comment by DevAlien** [comment]
But you could do this with the command, right? You mean that you would
select SSH, instead of custom command and you could set username and
password, right? It makes sense to have it

**Comment by rahulhaque** [comment]
Yes, I can do this now with the command. But the issue is two terminals open at the same time.

**Comment by DevAlien** [comment]
I will fix that in the following days :)
